beam/public/js/stock_entry_custom.js

```diff
@@ -100,21 +115,32 @@ async function show_handling_unit_recombine_dialog(frm) {
 	})
```

Comment:
On line 111, we should change this to 'extra-large'. [Frappe example](https://github.com/frappe/frappe/blob/b331163ce96188f81cf7f5d8a67aa4aab9f92637/frappe/public/js/frappe/utils/diffview.js#L62)

Reply:
changes has been done